wisp-launchdarkly/src/main/kotlin/wisp/launchdarkly/LaunchDarklyFeatureFlags.kt

```diff
@@ -222,8 +225,7 @@ class LaunchDarklyFeatureFlags constructor(
   private fun checkInitialized() {
     checkState(
       ldClient.isInitialized,
-      "LaunchDarkly feature flags not initialized. " +
-        "Did you forget to make your service depend on [FeatureFlags]?"
+      "LaunchDarkly feature flags not initialized."
```

Comment:
nit: Probably want specific advice still, e.g. have you called LaunchDarklyFeatureFlags.startUp() or bound a FeatureFlags ServiceModule (for misk).

Reply:
Wisp and Misk LD are completely separate (as in Misk is not using this code).  This is specific Wisp code now.